pom.xml

```diff
@@ -123,6 +124,12 @@
             <version>1.1</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>token-macro</artifactId>
```

Comment:
Why not make optional? Plugin lookup and method extraction is pretty simple and shown in pr plugin.